src/firefly/js/visualize/ui/FileUploadViewPanel.jsx

```diff
@@ -470,10 +419,11 @@ function ImageDisplayOption() {
     );
 }
 
-function UploadOptions({uploadSrc=FILE_ID, isloading, isWsUpdating, uploadKey}) {
+function UploadOptions({uploadSrc=FILE_ID, isLoading, isWsUpdating, uploadKey}) {
 
+    const [getUploadMetaInfo, setUploadMetaInfo]= useFieldGroupMetaState({isLoading:undefined, statusKey: undefined });
     const onLoading = (loading, statusKey) => {
-        dispatchComponentStateChange(panelKey, {isLoading: loading, statusKey:loading?statusKey:''});
+        setUploadMetaInfo({isLoading:loading, statusKey: loading?statusKey:''});
```

Comment:
to fix a bug, change to:
```js
 setUploadMetaInfo({...getUploadMetaInfo(), isLoading:loading, statusKey: loading?statusKey:''});
```
using `getUploadMetaInfo` makes sure the state is only modified not completely replaced.